prime-router/metadata/fhir_mapping/hl7/resource/MessageHeader.yml

```diff
@@ -78,3 +78,18 @@ extension_3:
         system: "http://terminology.hl7.org/CodeSystem/v3-AcknowledgementCondition"
       vars:
         code: MSH.16
+
+extension_4:
+  condition: $msh2 NOT_NULL && $msh2 NOT_EQUALS_STRING ^~\&
```

Comment:
Will these characters always appear in this order?

Reply:
It looks like this is the recommended set of characters from the [spec ](https://hl7-definition.caristix.com/v2/HL7v2.5.1/Fields/MSH.2), but it does not necessarily have to be those set of characters. 

The second part of this check is not strictly necessary as when msh2 is not populated, the FHIR->HL7 library will automatically default to `^~\&`

Reply:
Updated to specifically only look for `^~\&` when adding this to the bundle